### ISSUE TITLE
App is extremly slowing down with big cache (big media exchange)

### ISSUE THREAD

**Issue by KeenMaron** [element-hq/element-android#8726]
### Steps to reproduce

We have the problem, that if we exchange many pictures or videos in a room, the app cache grows and grows. After a size of 1-2 GB the app gets really slow and sending message is delayed for 2-4 seconds.
Clearing the cache makes the app fast again, but if the cache grows again it gets more and more unresponsive.

It is hard to reproduce after cleaning the cache.

Maybe it is related to:
https://github.com/element-hq/element-android/issues/8446

#### What did you expect?
Fast application.
#### What happened instead?
It is very slow with after a specific size of the cache.

### Your phone model

Pixel 8 pro

### Operating system version

Android

### Application version and app store

1.6.8

### Will you send logs?

No

### Are you willing to provide a PR?

No

**Comment by ainola** [comment]
Yeah, this one is killing me. I have about 30 rooms of various sizes and the cache grows large enough that I have to clear it weekly. Otherwise the performance gets unbearable: 30 second load times, for instance.
